readme/desktop.md

```diff
@@ -1,5 +1,103 @@
 # Desktop application
 
+
+
+For general information relevant to all the applications, see also [Joplin home page](https://joplinapp.org).
+
+# An illustrated Guide to Using Joplin.  
+•	Why note taking at all?  
+o	It makes it easier to find small bits of information that don’t warrant a full document: the VIN number and tire air pressure for your car; Veterinary data for pets, the phone number and ordering website for a good pizza place, all the way up to major projects of any type.
+•	Why Joplin over other note managers?  
+
+o	OneNote – Too many limitations, and too many of the wrong capabilities. Learning curve is high.
+
+o	EverNote – Too buggy.
+
+o	Google Keep. Too slow, no offline/desktop app. No way to nest notes. 
+
+o	Also, these above leave you beholden to a corporate data format which may or may not be in your best interest.
+o	Joplin features are straightforward: the features it has, are pretty bulletproof.  If a feature is included, it works and consistently.
+
+Joplin the name: 
+It comes from Scott, not Janice.  Scott Joplin is a favorite composer and pianist of the primary developer for the app.
+
 <img src="https://raw.githubusercontent.com/laurent22/joplin/dev/Assets/WebsiteAssets/images/DemoDesktop.png" style="width: 100%">
 
-For general information relevant to all the applications, see also [Joplin home page](https://joplinapp.org).
\ No newline at end of file
+![Editor Layout](https://user-images.githubusercontent.com/17282079/227723731-a3a9dde7-a0ca-406c-b081-5da30643f86b.jpg)
+
+# A note about the markdown editor for the unfamiliar:
+
+If you have ever worked on a wiki, you know what it looks like and how it works.  Plain text formatting marks are translated into more advanced formatting for display. For instance, adding highlighting, bulleted lists, italics, tables etc.  The markdown editor shows you the raw text it is translating and also gives you formatting buttons at the top.
+
+For those people who may remember the old WordPerfect “Reveal codes” button, which allowed you to directly change the text formatting behind the display, markdown works much the same way.
+
+Tip: You can teach yourself markdown by using the format buttons at the top and seeing both how markdown sets it up, and how it renders as RTF.
```

Comment:
If this is an introductory guide, people might not know what RTF means. So either use the explanation in brackets the first time tou use the abbreviation `RTF (Rich Text Format)` or remove it. There's no reason to say `renders as RTF`. Replace with `and how it is rendered.` - seems to be sufficient.

Reply:
Agreed.  Let's put it in that way.